cypress/support/commands/pageTestFragment/metaData.js

```diff
@@ -0,0 +1,197 @@
+Cypress.Commands.add('retrieveMetaDataContent', (metaDataTag, content) => {
+  cy.get(metaDataTag).should('have.attr', 'content', content);
+});
+
+Cypress.Commands.add('facebookMeta', (fbAdmins, appID, articleAuthor) => {
+  cy.get('head meta[name="fb:admins"]').should(
+    'have.attr',
+    'content',
+    fbAdmins,
+  );
+  cy.get('head meta[name="fb:app_id"]').should('have.attr', 'content', appID);
+  cy.get('head meta[name="article:author"]').should(
+    'have.attr',
+    'content',
+    articleAuthor,
+  );
+});
+
+Cypress.Commands.add('metadataAssertion', () => {
+  cy.window().then(win => {
+    cy.get('head meta[name="description"]').should(
```

Comment:
Same as above on the the head, I think they can be chained rather than so many cy.gets